news/_posts/releases/2022-03-07-prestashop-1-7-8-4-maintenance-release.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title:  "Release of PrestaShop 1.7.8.4"
+subtitle: "Maintenance version of the 1.7.8.x branch"
+date:   2022-03-07 18:00:00
+authors: [ PrestaShop ]
+image: /assets/images/2022/01/1.7.8.4_banner.jpg
+icon: icon-checked
+tags:
+ - version
+ - patch
+ - releases
+ - 1.7.8.x
+ - 1.7
+
+---
+
+PrestaShop 1.7.8.4 is now available. This maintenance release fixes 23 bugs and feature request.
+
+![1.7.8.4 is available!](/assets/images/2022/01/1.7.8.4_banner.jpg)
```

Comment:
```suggestion
![1.7.8.4 is available!](/assets/images/2022/03/1.7.8.4_banner.jpg)
```

Reply:
Final link has to be https://build.prestashop.com/assets/images/2022/03/1.7.8.4_banner.jpg